toolkit/tools/imagemodifier/main.go

```diff
@@ -0,0 +1,57 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+package main
```

Comment:
Generally we use `./toolkit/tools/<toolname>/<toolname>.go` four our tools.
Also, please add this too to `go_tool_list` in `tools.mk`. This will let the makefile build the tool if needed (you can do `make go-imagemodifier REBUILD_TOOLS=y` to build it, and if needed refer to it elsewhere in the makefiles with `$(go-imagemodifier)`.

Reply:
resolved the second comment, for the first comment, the imagecustomizer.go resides in ./toolkit/tools/pkg/imagecustomizerlib, and the main.go is in ./toolkit/tools/imagecustomizer, I thought this is by design, so the osmodifier follows suit.

Reply:
ok, not a big deal either way.